app/data/json_extraction.py

```diff
@@ -0,0 +1,12 @@
+import json
+
+
+def extract_data() -> dict:
+    with open("app/data/config.json", "r") as data_in:
+        return json.load(data_in)
+
+
+data = extract_data()
+FUEL_PRICE = data["FUEL_PRICE"]
+customers = data["customers"]
+shops = data["shops"]
```

Comment:
you can import this function in main and  create constants there, instead of importing them, that's more explicit way